src/main/java/org/threadly/examples/features/PrioritySchedulerExample.java

```diff
@@ -0,0 +1,153 @@
+      
+      @Override
+      public void run() {
+        /*
+         * This is a low priority task.
+         * Low priority is run before Starvable Priority tasks 
+         * and after High Priority tasks.
+         * Example: stats gathering for long running task
+         */
+        
+      }
+    }, TaskPriority.Low));
+    
+    futures.add(executor.submit(new Runnable() {
+
+      @Override
+      public void run() {
+        /*
+         * This is the High priority task that will be run.
+         * High priority tasks are the highest priority in the thread pool.
+         * Example use case: Servicing requests for the application
+         * or doing processing that requires results as quickly as possible
+         */
+        
+      } 
+     }, TaskPriority.High));
+    
+    //adding new high priority tasks to fill the pool
+    for (int i = 0; i < numThreads - 3; i++) {
+      futures.add(executor.submit(new HighPriorityTask(), TaskPriority.High));
+    }
+    //assigning to the static references for use in other methods
+    statFutures = futures;
+    
+    
+    return executor;
+  }
+  
+  /**
+   * gets the list of Listenable Futures to help with blocking. relies on getPriorityScheduler to be executed before this will return non-null
+   * @return -> a list of ListenableFutures, null if this has not been set
+   */
+  public static List<ListenableFuture<?>> getFutures() {
+    return statFutures;
+  }
+  
+  /**
+   * 
+   * @param executor -> a PriorityScheduler
+   * @return -> the same PriorityScheduler, but with the tasks started
+   */
+  public static PriorityScheduler startTasks(final PriorityScheduler executor) {
+    /*
+     * the prestartAllThreads() method can be called before or after
+     * tasks have been submitted
+     */
+    executor.prestartAllThreads();
+    return executor;
+  }
+  
+  /**
+   * 
+   * @param executor -> a PriorityScheduler
+   * @param futures -> the list of ListenableFutures
+   * @return -> the same PriorityScheduler, but after tasks have been executed
+   */
+  public static PriorityScheduler startTasksBlocking(final PriorityScheduler executor, final List<ListenableFuture<?>> futures) {
+    executor.prestartAllThreads();
+    try {
+      FutureUtils.blockTillAllCompleteOrFirstError(futures);
+    } catch (InterruptedException e) {
+      //should handle the InterruptedExeception
+      e.printStackTrace();
+    } catch (ExecutionException e) {
+      //should handle the ExecutionException
+      e.printStackTrace();
+    }
+    return executor;
+  }
+  
+  /**
+   * prevents any future tasks from being submitted to the PriorityScheduler and does not run queued tasks
+   * @param executor -> the PriorityScheduler to operate on
+   * @param recurringStop -> whether or not to stop recurring tasks as well
+   */
+  public static void disposePrioritySchedulerTasks(final PriorityScheduler executor, boolean recurringStop) {
+    if (recurringStop) {
+      executor.shutdownNow();
```

Comment:
I think what you meant to do here here is remove the recurring task.  In order to do that you should make this non-static, and save a reference to the runnable your constructing above (and submitting at a fixed rate).  You should then call .remove(Runnable) to prevent future executions.